virtcontainers/device/config/config.go

```diff
@@ -75,6 +83,12 @@ var SysDevPrefix = "/sys/dev"
 // SysIOMMUPath is static string of /sys/kernel/iommu_groups
 var SysIOMMUPath = "/sys/kernel/iommu_groups"
 
+// VhostUserSockDir is static string of vhost-user socket directory
+var VhostUserSockDir = "/var/run/kata-containers/vhost-user-socket"
+
+// VhostUserSockDir is static string of emulated vhost-user block node directory
+var VhostUserMknodDir = "/var/run/kata-containers/vhost-user-mknod"
```

Comment:
Curious if it's there is a way to avoid having hard-coded paths here.

Isn't that something that we might want to overload from a `configuration.toml` file later? I'm a bit new to this code, but it looks to me like `config-settings.go` has a number of these paths defined. I would put yours there as well.